src/app.js

```diff
@@ -1 +1,186 @@
-console.log('Hallo daar!');
\ No newline at end of file
+console.log('Hallo daar!');
+import axios from 'axios';
+
+const countryList = document.getElementById("country-List");
+const URI = "https://restcountries.com/v2/";
+const ENDPOINT = "all";
```

Comment:
Variabelen in camelCase schrijven
```suggestion
const uri = "https://restcountries.com/v2/";
const endpoint = "all";
```